### ISSUE TITLE
3440x1440

### ISSUE THREAD

**Issue by FinderProcess** [sayoui001/ApexAHK-Reduce-recoil#15]
Hey. I was wonder if could add this resolution "3440x1440" to the script it's my native rez and nothing else work for me?

**Comment by mocchimon** [comment]
Try(Generated from Mgsweet's resolution generator): 
weapon1 = "2725,1384"
weapon2 = "3268,1381"
r99 = "2877,1314,1,2993,1298,0,2940,1338,1"
r301 = "2965,1301,0,3015,1290,1,3031,1298,1"
re45 = "2875,1300,1,2934,1306,0,2977,1338,1"
p2020 = "2882,1293,1,2925,1308,0,2956,1338,1"
g7 = "2818,1298,1,2972,1308,0,3051,1318,1"
flatline = "2958,1313,0,2821,1306,1,2841,1312,1"
prowler = "2879,1321,1,2924,1313,0,2915,1324,1"
hemlok = "2906,1293,1,2949,1312,0,3015,1298,1"
rampage = "2795,1300,1,2947,1313,0,3036,1310,1"
wingman = "2872,1312,1,2945,1310,0,2968,1334,1"
lstar = "2843,1297,1,2940,1318,0,2986,1292,1"
devotion = "3045,1294,1,2977,1306,0,2796,1296,1"
volt = "2945,1308,0,2839,1301,1,3010,1294,1"
havoc = "2967,1328,1,2970,1313,0,2932,1282,1"
spitfire = "3033,1296,1,2959,1318,1,2947,1282,1"
alternator = "2893,1305,1,2941,1306,1,2949,1304,0"
car = "2875,1293,1,2841,1297,1,2875,1294,1"
p3030 = "2906,1300,1,3049,1308,0,3002,1317,0"
havoc_turbocharger = "2904,1341"
devotion_turbocharger = "2956,1342"

**Comment by FinderProcess** [comment]
Yeah i have tried this before but dosent work for me it's only resolution i have a problem with and its my nativ cant get it to work
